UPGRADING.md

```diff
@@ -49,9 +49,7 @@ You can now install Bugsnag using the [Unity Package Manager](https://docs.unity
 
 All the files in the V6 package are located in a parent Bugsnag directory and so you will need to remove all the v5 files from your project to avoid having multiple instances of Bugsnag running.
 
-To remove these files, you can download and run the bash script `utils/remove-bugsnag-v5.sh` in the `bugsnag-unity` repository, passing the path to your project as a parameter.
-
-If you wish to do it manually, please remove the following directories and files:
+Please remove the following directories and files:
```

Comment:
I made this change because the script now seems unnecessary. it's very simple to just remove the files listed, and the script we provided feels like clutter to me. The instructions for upgrading are still there and very clear/simple, i just removed the script part of it.

Reply:
OK - seems reasonable.